### DISCUSSION TITLE
为什么PCL II不添加一个赞助主题同步功能？

### DISCUSSION THREAD

**Discussion by HexianTao9998** [Meloong-Git/PCL#2155] [提问] [answered]
首先，我有两台电脑，一台在学习宿舍，一台在家，都是赞助版的，但是有个我觉得非常重要的功能就是那些主题，它就是不能同步，比如说我在我家电脑解锁了铁杆粉，但在宿舍的那台就没有解锁;所以我希望龙腾猫跃可以解决这个问题。
我可以想几个想法：
1.通过QQ登录来同步主题（希望用这个）
2.通过检测是否加群来同步主题
3.通过微软登录来实现主题
4.通过邮箱注册登录来同步主题
5不要用手机验证码来同步主题（因为我没有手机）

希望龙腾猫跃加入我的提议

**Comment by MoYuan-CN** [comment]
见 #760

**Reply by HexianTao9998** [reply]
所以就没有解决办法了吗

**Comment by WForst-Breeze** [comment, marked as answer]
见 #760 ，如果可以导出、同步，那么只要有一个人解锁了所有主题，所有人都可以轻松通过导入和同步来解锁所有主题了。不论是上述的哪种方式，都不能完全保护用户信息和隐藏主题信息，总不能只有登录QQ、登录微软账户的时候才能使用隐藏主题吧（

**Reply by HexianTao9998** [reply]
那你说怎么办嘛！！苦恼

**Reply by WForst-Breeze** [reply]
目前没有别的解决办法，本人也是换了好几次设备，也都只能重新解锁，秋怡金和活跃橙可以联系龙猫重新获取，其他主题除了欧皇彩和铁杆粉都比较好解锁，但也只能这样了 :(

**Reply by MoYuan-CN** [reply]
欧皇彩也好办,用窗口卡位置就行
铁杆粉真的麻烦

**Reply by iPodtouch6557** [reply]
复制setup.ini行不行？
铁杆粉我是在买内测之前就已经启动了150多次了，现在启动了刚好375次，而且不是刷出来的（（（

**Reply by WForst-Breeze** [reply]
隐藏主题的解锁信息是存在注册表里的，复制 setup.ini 不行（

**Comment by iPodtouch6557** [comment]
其实如果不能靠文件来同步的话，你的这个想法还能用账号云存储功能实现，但那是不可能的tzg……
